### DISCUSSION TITLE
Login page for go_router

### DISCUSSION THREAD

**Discussion by longfit** [bdlukaa/fluent_ui#767] [General]
I am very interested in your 4.4.1 example using go_router.

my project need  loginPage for authenticating.
if user name and password is correctly below 

  
       FluentPageRoute(builder: (context) => const MyHomePage()),

Since I don't know how to modify your example how to
redirect or route MyHomePage() when login is successfully.

I am very happy if you give me some advice 

Best Regards

**Comment by bdlukaa** [comment]
The example app makes use of `go_router`'s `ShellRoute`. The "Flutter Package of the Week" video about it explains better how it works.

Then, the example app calculates what's the index of the current route

**Comment by longfit** [comment]
I have development login page based on go_router example, 

current result is shown as paneitem  page, I want to login page as individual scaffoldPage and jump into main page if
sign in ok

Have any suggestion to make use of go_router + fluent_ui

[Screencast from 2023-10-01 01-32-39.webm](https://github.com/bdlukaa/fluent_ui/assets/4157794/8da21bc9-69d2-4ed1-808c-7a901b53f0f1)
